packages/atlas/src/components/_video/VideoPlayer/VideoPlayer.styles.ts

```diff
@@ -414,7 +414,7 @@ export const Container = styled.div<ContainerProps>`
         > div {
           background-color: ${cVar('colorBackgroundOverlay')} !important;
           font: ${cVar('typographyDesktopT300Strong')} !important;
-          font-size: 1em !important;
+          font-size: ${({ isFullScreen }) => (isFullScreen ? '0.64em' : '16px')} !important;
```

Comment:
subtitles are way too big on fullscreen. I think we should stick to 32px in this case. 

You also didn't handle the size for mobile phones. The size of the subtitles should be 12px. Not sure about fullscreen(android users), but I would keep 12px as well. 

Btw, do these subtitles work on ios on fullscreen?

Reply:
@drillprop yup, they work properly on ios

Reply:
However they don't have proper font-family, but I'm affraid we can't do much about it